src/factorFinder/FactorComputeRequest.java

```diff
@@ -1,4 +1,7 @@
+
 package com.example.factorfinder;
+import com.example.factorfinder.InputConfig;

```

Comment:
Its good that yall figured out this was one of the reasons why the gradle build kept saying it could find the specific files.